java/http/java-http-consumer/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+USER root
+
+RUN microdnf update \
+    && microdnf install java-17-openjdk-headless shadow-utils \
```

Comment:
You should install the Java as we do here to get rid of some CVEs in the images. Both here as well as in the other Dockerfile: https://github.com/strimzi/strimzi-kafka-operator/blob/main/docker-images/base/Dockerfile#L11

Reply:
Ok I will change this way but it seems we need to do the same for the other examples, referring to the Java Kafka ones. There is the same problem there as well. But not for this PR.